package com.user.app.exception;


public class UserNotFoundException extends RuntimeException{
	
	public UserNotFoundException() {
		
	}
	
	public UserNotFoundException(String userName) {
		super("User already exists with username: "+ username);
	}

}
